frontend/src/scenes/actions/ActionEdit.js

```diff
@@ -187,30 +157,33 @@ export class ActionEdit extends Component {
                         </>
                     ))}
 
-                    <br />
-
-                    {this.state.saved && !isEditor && (
-                        <p className="text-success">Action saved.</p>
+                    {!isEditor && this.state.showSlackCheckbox ? (
+                        <div style={{ marginTop: 20, marginBottom: 15 }}>
+                            <label>
+                                <input
+                                    type="checkbox"
+                                    onChange={e => {
+                                        this.setState({ action: { ...action, post_to_slack: e.target.checked } })
+                                    }}
+                                    checked={action.post_to_slack}
+                                />
+                                &nbsp;Post to Slack when this action is triggered
```

Comment:
Could we have a `<small>configure</small>` link here to /setup (maybe even with ref) and show it even if the team doesn't have Slack configured? Maybe greyed out (opacity 0.7 or something) and input disabled

Reply:
done